### ISSUE TITLE
Version 3.0 does not get accepted bij Seirra Mail.app version 10.2

### ISSUE THREAD

**Issue by gbsom** [UniversalMailer/UniversalMailer#94]
Version 3.0 does not get accepted bij Seirra Mail.app version 10.2. Mail continues without the plugin.

<img width="413" alt="screenshot" src="https://cloud.githubusercontent.com/assets/26165294/23551614/e082cd10-0017-11e7-9157-88a1a86ec165.png">

Mail Plugin Manager does not here ! Plugjn is shown as enabled. But can't disable or delete it.

**Comment by UniversalMailer** [comment]
Every time Apple updates Mail.app you'll have to run the installer again. The plugin is still compatible with the latest version of macOS Sierra (10.12.4 as of today).

**Comment by gbsom** [comment]
Hi

Thanks for the response… but even after a running the installer the issue remains.

Kind regards, met de vriendelijke groeten,

Ger Somhorst
ger@somhorst.net
06-50646475

**Comment by UniversalMailer** [comment]
Try removing the plugin (as suggested here: http://universalmailer.github.io/UniversalMailer/faq.html) and installing it again, I can confirm it is working as expected on the latest version of macOS (I'm using it myself)

**Comment by gbsom** [comment]
Hi, 

Sorry for bothering you with this.

Tried the removal and de reinstallation after I made sure that I had the latest versions of everything:





The result remains unfortunate:




This is so a pitty whilst this tool is so needed. Not working on my MacBookPro also prevents to migrate to Sierra on my iMac.

Any suggestions ?


Kind regards, met de vriendelijke groeten,

Ger Somhorst
ger@somhorst.net
06-50646475
